src/surfing_penguin/extensions/__init__.py

```diff
@@ -19,5 +19,14 @@ def init_app(app):
         app.config['SQLALCHEMY_DATABASE_URI'],
         echo=False)
     from src.surfing_penguin import models  # noqa: F401
+    from src.surfing_penguin.models import User  # noqa: F401
     Base.metadata.create_all(db_engine)
     session.__init__(bind=db_engine)
+    ADMIN_NAME = app.config['ADMIN_NAME']
+    ADMIN_PASSWORD = app.config['ADMIN_PASSWORD']
+    if session.query(User).filter_by(username=ADMIN_NAME).first() is None:
+        new_user = User(ADMIN_NAME, ADMIN_PASSWORD)
+        new_user.user_role = 'admin'
+        new_user.id = session.query(User).count() + 1
+        session.add(new_user)
+        session.commit()
```

Comment:
Please add a # TODO here. I think there would be a better way than creating the admin user while initialization, but for now let's leave it there.